HW_06/Task2/Program.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace ConsoleApp6
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string verse = "Текст это упорядоченный набор предложений предназначенный для того чтобы выразить некий смысл";
+
+            DeleteWordMax(verse);
+            Console.WriteLine();
+
+            ChangePosition(verse);
+            Console.WriteLine();
+
+            CountLetterMark(verse);
+
+            ReversWords(verse);
+        }
+
+        static void DeleteWordMax(string text)
+        {
+            char[] symbol = { ' ', '.', ',', '!', '?', ';', ':', '"', '-', '\'', '\\' };
```

Comment:
i see the same array in each method. this is a copy past logic. I would suggest to move this as a field